httpref.go

```diff
@@ -40,7 +41,13 @@ func (r References) InRange(code string) References {
 	for _, v := range r {
 		if strings.Contains(v.Name, "-") {
 			parts := strings.Split(v.Name, "-")
-			if code >= parts[0] && code <= parts[len(parts)-1] {
+			if len(parts) != 2 {
+				fmt.Printf("Invalid range format for reference name: %s\n", v.Name)
+				continue
+			}
```

Comment:
We can probably skip this as the tests will catch this problem - the command will crash in the acceptance tests.

Reply:
> We can probably skip this as the tests will catch this problem - the command will crash in the acceptance tests.

well I will remove this from httpref.go